Makefile

```diff
@@ -53,6 +53,7 @@ verify: build
 	hack/verify-govet.sh
 	hack/verify-generated-deep-copies.sh
 	hack/verify-generated-conversions.sh
+	hack/verify-generated-clientsets.sh
```

Comment:
You need to add `hack/update-generated-sclientsets.sh` to `update` target as well.